Xamarin.Android-Tests.sln

```diff
@@ -49,7 +51,15 @@ Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "Xamarin.Forms.Performance.I
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Xamarin.Forms.Performance.Integration.Droid", "tests\Xamarin.Forms-Performance-Integration\Droid\Xamarin.Forms.Performance.Integration.Droid.csproj", "{576312CC-83FF-48B1-A473-488CDC7121AD}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Xamarin.Android.Bcl-Tests", "tests\Xamarin.Android.Bcl-Tests\Xamarin.Android.Bcl-Tests.csproj", "{E865C28E-32AF-4210-A41D-5791C39A9D85}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Xamarin.Android.Bcl-Tests", "tests\Xamarin.Android.Bcl-Tests\Xamarin.Android.Bcl-Tests\Xamarin.Android.Bcl-Tests.csproj", "{04D4E45E-0EBE-40F7-B170-0A8F4D74DCBC}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TestRunner.Core", "tests\Xamarin.Android.Bcl-Tests\TestRunner.Core\TestRunner.Core.csproj", "{3CC4E384-4985-4D93-A34C-73F69A379FA7}"
```

Comment:
Perhaps the `TestRunner.Core` project should be moved "up" a level? (This would be particularly useful if we ever get any "takers" to migrate our existing NUnit tests to xUnit!)

I think this could be `tests/TestRunner.Core`, etc.

Reply:
I moved the runner backend assemblies to `test` and renamed `tests/Xamarin.Android.Bcl-Tests` to `tests/BCL-Tests` because we have not one but two projects inside that dir and don't think we want to pollute `tests/` with these two (since `LocalTests.NUnit` is not an app, but just an assembly with tests unrelated to and unused by the other projects in `tests/`)